src/tcgwars/logic/impl/gen3/UnseenForces.groovy

```diff
@@ -2499,12 +2508,20 @@ public enum UnseenForces implements LogicCardInfo {
         weakness L
         pokeBody "Overpowering Fang", {
           text "As long as Feraligatr ex is your Active Pokémon, each player's Pokémon (excluding Pokémon-ex) can't use any Poké-Powers or Poké-Bodies."
-          getterA (IS_ABILITY_BLOCKED) { Holder h ->
-            if (self.active && !h.effect.target.EX) {
-              if (h.effect.ability instanceof PokePower || h.effect.ability instanceof PokeBody) {
-                h.object=true
+          def eff
+          onActivate{
+            eff = getter (IS_ABILITY_BLOCKED) { Holder h ->
+              if (self.active && !h.effect.target.EX) {
+                if (h.effect.ability instanceof PokePower || h.effect.ability instanceof PokeBody) {
+                  h.object=true
+                }
               }
             }
+            new CheckAbilities().run(bg)
+          }
+          onDeactivate {
+            eff.unregister()
+            new CheckAbilities().run(bg)
```

Comment:
actually, `getterA`,  `onActivate` and `onDeactivate` can all coexist in a single ability definition. @starg09

Reply:
Oh? I stand corrected then. Will revert this later, sorry!

Reply:
The order of run is: 
- actionA registered [only a single one allowed]
- delayedA runs [only a single one allowed]
- getterA's run [multiples allowed]
- onActivate called